<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>271</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="Home_btn">
       <property name="text">
        <string>Home</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Menu_btn">
       <property name="text">
        <string>Menu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Setting_btn">
       <property name="text">
        <string>Setting</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Lock_btn">
       <property name="text">
        <string>Lock</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FIRE ALARM CONTROL SYSTEM</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="Main_stk">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>461</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>48</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="Main_TW">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>441</width>
        <height>371</height>
       </rect>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Number</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Location</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>State</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="Acknowledge_btn">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>420</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>ACK</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>441</width>
        <height>331</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="Isolation_btn">
         <property name="text">
          <string>Isolation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Log_btn">
         <property name="text">
          <string>Log</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Test_btn">
         <property name="text">
          <string>Test</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Download_Log_btn">
         <property name="text">
          <string>Download Log</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Menu</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>48</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Setting</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>441</width>
        <height>331</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="Setting_lvl1_btn">
         <property name="text">
          <string>Security Level 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Setting_lvl2_btn">
         <property name="text">
          <string>Security Level 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Setting_lvl3_btn">
         <property name="text">
          <string>Security Level 3</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>411</width>
        <height>371</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="Zone1_isol_lbl">
         <property name="text">
          <string>Normal</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QPushButton" name="Zone2_isol_btn">
         <property name="text">
          <string>Zone 2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="Zone2_isol_lbl">
         <property name="text">
          <string>Normal</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="Zone3_isol_btn">
         <property name="text">
          <string>Zone 3</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="Zone1_isol_btn">
         <property name="text">
          <string>Zone 1</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QPushButton" name="Zone4_isol_btn">
         <property name="text">
          <string>Zone 4</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="Zone4_isol_lbl">
         <property name="text">
          <string>Normal</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="Zone3_isol_lbl">
         <property name="text">
          <string>Normal</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Isolation</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>48</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Test</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>401</width>
        <height>401</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QPushButton" name="Zone4_test_btn">
         <property name="text">
          <string>Zone 4</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="Zone3_test_btn">
         <property name="text">
          <string>Zone 3</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QPushButton" name="Zone2_test_btn">
         <property name="text">
          <string>Zone 2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="Zone1_test_btn">
         <property name="text">
          <string>Zone 1</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QTableWidget" name="log_TW">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>441</width>
        <height>371</height>
       </rect>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Number</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Location</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>State</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>48</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Log</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="Time_stamp_lbl">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Time YYYY - MM - DD - hh : mm : ss</string>
    </property>
   </widget>
   <widget class="QLabel" name="AlarmColor">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Lock_Status_lbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Lock_status : Unlock</string>
    </property>
   </widget>
   <widget class="QLabel" name="Security_level_lbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Security Level : Level 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lock_lbl">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../일지 gif파일/lock.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
